examples/telemetry_channel_metrics.lua

```diff
@@ -0,0 +1,73 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+Monitor telemetry traffic by channel
+
+*Example Heka Configuration*
+
+.. code-block:: ini
+
+    [ByChannelHour]
+    type = "SandboxFilter"
+    ticker_interval = 10
+    preserve_data = false
+    message_matcher = "Logger == 'telemetry'"
```

Comment:
You this is incorrect.  You will never get telemetry.error L56 and you will also process all the saved sessions.   FYI,  the updated executives summary ping will have all the information you need, so you can switch to that shortly (will land today).